Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM ubuntu:xenial
-
-RUN apt-get -y update \
-    && apt-get -y install \
-    curl wget
+FROM alpine:3.6
 
+ENV ALPINE_GLIBC_VERSION 2.26-r0
 ENV HELM_VERSION 2.6.2
 
+RUN apk update && apk --no-cache add ca-certificates openssl curl wget git bash \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
```

Comment:
Why do we need to update glibc?

Reply:
alpine doesn't provide glibc at all. Here we install it, assuming that the `draughtsman` binary will need it to run.

Reply:
Does it? Do statically compiled binaries still require `glibc`?